src/client/src/containers/RightSidebar/index.tsx

```diff
@@ -64,8 +66,10 @@ interface IRightSidebarProps {
 }
 
 interface IRightSidebarState {
-  frontendDropdownValue?: ISelected | undefined;
-  items: string[];
+  sideBarOpen: boolean;
+  // Since componentWillReceiveProps lifecycle method is now depricated and we don't have acces to prevProps in getDerivedStateFromProps,
```

Comment:
NIT: type `deprecated`

Reply:
It arguably should be part of the state and not props anyways since it is only part of the rightsidebar

Reply:
This comment is for the locationPathname string. I will make it clearer :)

Reply:
After recent changes, it doesn't exist anymore :)